<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditWidget</class>
 <widget class="QWidget" name="TextEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>По умолчанию</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Марк. список (диск)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Марк. список (окружность)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Марк. список (квадрат)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нумер. список (цифры)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нумер. список (стр. буквы)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нумер. список (загл. буквы)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="comboFont">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboSize">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonBold">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="qrc.qrc">
           <normaloff>:/icon/icons/textbold.png</normaloff>:/icon/icons/textbold.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonUnderline">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="qrc.qrc">
           <normaloff>:/icon/icons/textunder.png</normaloff>:/icon/icons/textunder.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonItalic">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="qrc.qrc">
           <normaloff>:/icon/icons/textitalic.png</normaloff>:/icon/icons/textitalic.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAlignJustify">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="qrc.qrc">
           <normaloff>:/icon/icons/textjustify.png</normaloff>:/icon/icons/textjustify.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroupAlign</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAlignLeft">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="qrc.qrc">
           <normaloff>:/icon/icons/textleft.png</normaloff>:/icon/icons/textleft.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroupAlign</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAlignCenter">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="qrc.qrc">
           <normaloff>:/icon/icons/textcenter.png</normaloff>:/icon/icons/textcenter.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroupAlign</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAlignRight">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="qrc.qrc">
           <normaloff>:/icon/icons/textright.png</normaloff>:/icon/icons/textright.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroupAlign</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupAlign"/>
 </buttongroups>
</ui>
